site/zenodo_rdm/assets/semantic-ui/less/zenodo_rdm/previewer/mirador-previewer.less

```
@@ -22,6 +23,12 @@
     box-shadow: 0px 0px 0px 1px #A9D5DE inset, 0px 0px 0px 0px rgba(0, 0, 0, 0);
 }
 
+.image-container {
```

Maybe using a name which suggest that it's not related to mirador?

```suggestion
.simple-image-container {
```